contributing.md

```diff
@@ -0,0 +1,39 @@
+# How to Contribute
+
+Thanks for your interest in contributing to `Markdig`! Here are a few general guidelines on contributing and
+reporting bugs that we ask you to review. Following these guidelines helps to communicate that you respect the time of
+the contributors managing and developing this open source project. In return, they should reciprocate that respect in
+addressing your issue, assessing changes, and helping you finalize your pull requests. In that spirit of mutual respect,
+we endeavour to review incoming issues and pull requests within 10 days, and will close any lingering issues or pull
+requests after 60 days of inactivity.
```

Comment:
```suggestion
Thanks for your interest in contributing to `Markdig`! Here are a few general guidelines on contributing and
reporting bugs that we ask you to review. Following these guidelines helps to communicate that you respect the time of
the contributors managing and developing this open source project.
```

Removing the "in return" and the "x days". There is no way I will commit to any constraints regarding any OSS repository I maintain.